machine.go

```diff
@@ -204,6 +204,37 @@ func (cfg *Config) Validate() error {
 	return nil
 }
 
+func (cfg *Config) ValidateLoadSnapshot() error {
+	if cfg.DisableValidation {
+		return nil
+	}
+
+	for _, drive := range cfg.Drives {
+		if BoolValue(drive.IsRootDevice) {
+			rootPath := StringValue(drive.PathOnHost)
+			if _, err := os.Stat(rootPath); err != nil {
+				return fmt.Errorf("failed to stat host drive path, %q: %v", rootPath, err)
+			}
+
+			break
+		}
+	}
```

Comment:
It is interesting we validate any drives the user passes in via configuration but none of the handlers which run in a load snapshot do anything with the drives.

Reply:
None of them do, but if the drives are not in place, the LoadSnapshot call will fail, so I think it is right to validate this.